### DISCUSSION TITLE
Can i use the enter key to auto complete commands?

### DISCUSSION THREAD

**Discussion by celty3** [thebaselab/codeapp#1001] [Q&A]
The question may sound weird as I’m not well versed in the technical words used to describe coding
What i want to ask is, can i use the enter key to autocomplete commands such as ‘class’ or ‘border-color’ so that the app auto completes them for me, like in vscode?

**Comment by bummoblizard** [comment]
Yes. It's only available in CSS files though.

![image](https://github.com/thebaselab/codeapp/assets/38398443/e0c9efe1-60c3-44d7-8625-19afe45cd2c3)
